workflow/scripts/analysis/predictive_signature_testing.py

```
@@ -0,0 +1,161 @@
+    # ### Bootstrap to get confidence intervals
+    sampled_cindex_bootstrap = []
+
+    hazards_and_outcomes = pd.DataFrame({
+        'hazards': feature_hazards,
+        'survival_event_binary': clinical_data_subset['survival_event_binary'],
+        'survival_time_years': clinical_data_subset['survival_time_years']
+    }, index=clinical_data_subset.index)
+
+    bootstrap_count = 1000
+
+    for idx in range(bootstrap_count):
```

Ruff (0.8.2)

Loop control variable `idx` not used within loop body

Rename unused `idx` to `_idx`

(B007)